src/frontend/src/components/RecordSearch/Record/RecordSummary/CaseBalances.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import BalanceList from "./BalanceList";
+import { CaseData } from "../types";
+import currencyFormat from "../../../../service/currency-format";
+
+interface Props {
+  cases?: CaseData[];
+  totalBalance: number;
+}
+
+const hasBalanceDue = (element: CaseData) => {
+  return element.balance_due > 0;
+};
+
+export default class CaseBalances extends React.Component<Props> {
+  render() {
+    if (!this.props.cases) {
+      return null;
+    }
+
+    const listItems = this.props.cases
+      .filter(hasBalanceDue)
+      .map((element: CaseData, i: number) => {
+        return (
+          <li className="mb2" key={i}>
```

Comment:
I think we can use the `case_number` as the `key` here.

Reply:
Good catch, updated to use the `case_number` as the key